test/controllers/SendYourFileControllerSpec.scala

```diff
@@ -332,6 +335,64 @@ class SendYourFileControllerSpec extends SpecBase {
         }
       }
 
+      "must return NoContent when the file status is 'RejectedSDES'" in {
```

Comment:
Can you change the name of this test please, because that doesn't represent what the test is actually doing. It should be something like "must return OK and load the 'FileProblem' page when the file status is 'RejectedSDES'"